haskell/src/SwiftNav/SBP/Acquisition.hs

```diff
@@ -0,0 +1,26 @@
+module SwiftNav.SBP.Acquisition where
```

Comment:
I exported everything because there was nothing no to export, but might be better form to export everything anyway? What would the `Data.HashMap` be for? For a message type to message map?